adr/ADR-00.md

```diff
@@ -0,0 +1,51 @@
+# Metadata for Stream and Consumer
+
+|Metadata|Value|
+|--------|-----|
+|Date    |2023-01-23|
+|Author  |@Jarema|
+|Status  |Proposed|
```

Comment:
Make this `Approved`, once merged it will be approved and we know we already have a PR in the server, Proposed only really for stuff thats just exploring ideas and with no immediate hopes for implementation